<launch>
	<arg name="world_object_topic" default="/world_object" />

	<node pkg="kut_ugv_sensor_fusion" type="traffic_sign_detector_node" name="traffic_sign_detector_node" output="screen" >
	<param name="camera_topic" type="string" value="/mono_cam/image_rect_color" />
	<param name="laser_objects_topic" type="string" value="/bumper_laser/scan/objects" />
	<param name="svm_model_url" type="string" value="$(find kut_ugv_sensor_fusion)/calib/traffic_sign_svm.yaml" />
	<param name="calibration_url" type="string" value="$(find kut_ugv_sensor_fusion)/calib/calib.yaml" />
	<param name="output" type="string" value="$(arg world_object_topic)" />
	</node>
</launch>
